source/battle.py

```diff
@@ -538,23 +601,15 @@ def fight(
                     if remaining_health_bars_enemy > 20:
                         remaining_health_bars_enemy = 20
 
-                    cout(f"PLAYER: {player_health} / {player_max_health}\n")
-                    cout(
-                        f"|{health_color}{remaining_health_bars * remaining_health_symbol}" +
-                        f"{lost_health_bars * lost_health_symbol}{color_default}|\n"
-                    )
-                    cout(f"ENEMY: {enemy_health} / {enemy_max_health}\n")
-                    cout(
-                        f"|{health_color_enemy}{remaining_health_bars_enemy * remaining_health_symbol}" +
-                        f"{lost_health_bars_enemy * lost_health_symbol}{color_default}|"
-                    )
                     player["xp"] += enemy_max * enemy_max_damage / 3
                     if player["current mount"] in player["mounts"]:
                         player["mounts"][player["current mount"]]["level"] += round(random.uniform(.05, .20), 3)
                     player["health"] += random.randint(0, 3)
                     enemies_remaining -= 1
+                    cout("You killed one {enemy_singular}!")
```

Comment:
```suggestion
                    cout("You killed text_handling.a_an_check(enemy_singular)!")
```
Just a small suggestion

Reply:
You've forgotten to put the function in '{' brackets! Could you edit that so I commit it then?

Reply:
`cout()` works like print then?
I fixed my suggestion.

Reply:
> `cout()` works like print then? I fixed my suggestion.

Yes, it's just a custom function to print stuff.